DFC.ServiceTaxonomy.GraphSync/GraphSyncers/Fields/BooleanFieldGraphSyncer.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Threading.Tasks;
+using DFC.ServiceTaxonomy.GraphSync.Extensions;
+using DFC.ServiceTaxonomy.GraphSync.GraphSyncers.Interfaces;
+using Newtonsoft.Json.Linq;
+
+namespace DFC.ServiceTaxonomy.GraphSync.GraphSyncers.Fields
+{
+    public class BooleanFieldGraphSyncer : IContentFieldGraphSyncer
```

Comment:
Would you mind adding an unit test for this?
Shouldn't take too long, mostly cut and pasting.

Reply:
Yep will do!